proposals/csharp-11.0/low-level-struct-improvements.md

```diff
@@ -934,20 +934,31 @@ The lifetime parameter `$this` on type definitions is _not_ predefined but it do
 
 For brevity sake a type which has no explicit lifetime parameters treated as if there is `out $this` defined and applied to all fields of the type. A type with a `ref` field must define explicit lifetime parameters.
 
-These rules exists to support our existing invariant that `T` can be assigned to `scoped T` for all types. That maps down to `T<$a, ...>` being assignable to `T<$local, ...>` for all lifetimes known to wider than ype of `$local`. Further this supports other items like being able to assign `Span<T>` from the heap to those on the stack. This does exclude types where fields have differing lifetimes for non-ref values but that is the reality of C# today. Changing that would require a significant change of C# rules that would need to be mapped out. 
+These rules exists to support our existing invariant that `T` can be assigned to `scoped T` for all types. That maps down to `T<$a, ...>` being assignable to `T<$local, ...>` for all lifetimes known to be convertible to `$local`. Further this supports other items like being able to assign `Span<T>` from the heap to those on the stack. This does exclude types where fields have differing lifetimes for non-ref values but that is the reality of C# today. Changing that would require a significant change of C# rules that would need to be mapped out. 
 
 The type of `this` for a type `S<out $this, ...>` inside an instance method is implicitly defined as the following:
-- For normal instance method: `ref<$local> S<$ro, ...>`
-- For instance method annotated with `[UnscopedRef]`: `ref<$ro> S<$ro, ...>`
-The lack of an explicit `this` parameter forces the implicit rules here. For complex samples and discussions likely better to use an explicit parameter.
+- For normal instance method: `ref<$local> S<$cm, ...>`
+- For instance method annotated with `[UnscopedRef]`: `ref<$ro> S<$cm, ...>`
+The lack of an explicit `this` parameter forces the implicit rules here. For complex samples and discussions consider writing as a `static` method and making `this` an explicit parameter.
+
+```csharp
+ref struct S<out $this>
+{
+    // Implicit this can make discussion confusing 
+    void M<$ro, $cm>(ref<$ro> S<$cm> s) {  }
+
+    // Rewrite as explicit this to make discussion require less memory
```

Comment:
Does "memory" refer to the reader's memory? If so, perhaps reword to reduce ambiguity:
```suggestion
    // Rewrite as explicit this to simplify discussion
```